modules/ingress-operator.adoc

```diff
@@ -20,9 +20,12 @@ ifdef::cluster-caps[= Ingress Capability]
 == Purpose
 
 ifdef::cluster-caps[]
+The Ingress Operator provides the features for the `Ingress` capability. Ingress capability is enabled by default.
```

Comment:
```suggestion
The Ingress Operator provides the features for the `Ingress` capability. The Ingress capability is enabled by default.
```
In the first sentence, does "Ingress" need to be formatted in backticks? I don't see it formatted that way anywhere else on this page.

Reply:
Good point. I removed the backticks.